src/TimeZones.jl

```diff
@@ -3,6 +3,7 @@ module TimeZones
 using Dates
 using Printf
 using Serialization
+using RecipesBase
```

Comment:
I want to move towards only importing what is being used:
```suggestion
using RecipesBase: @receipe
```